DuckDuckGo/NetworkProtectionRootView.swift

```diff
@@ -20,25 +20,32 @@
 #if NETWORK_PROTECTION
 
 import SwiftUI
+import NetworkProtection
 
 struct NetworkProtectionRootView: View {
     let model = NetworkProtectionRootViewModel()
+    let inviteCompletion: () -> Void
 
     var body: some View {
+        let inviteViewModel = NetworkProtectionInviteViewModel(
```

Comment:
This is not idea for now. It would be good to use a more elegant DI mechanism than letting the View do it. But for now, this will do. Once the rest of the feature is more fleshed out, I’ll tidy this up.

Reply:
What a "DI mechanism"?

Reply:
dependency injection (updated original comment)